spacy_transformers/layers/listener.py

```diff
@@ -58,16 +58,32 @@ def verify_inputs(self, inputs):
 
 def forward(model: TransformerListener, docs, is_train):
     if is_train:
-        model.verify_inputs(docs)
-        return model._outputs, model.backprop_and_clear
+        # This might occur during training when the tok2vec layer is frozen / hasn't been updated.
+        # In that case, it should be set to "annotating" so we can retrieve the embeddings from the doc.
+        if model._batch_id is None:
+            outputs = []
+            for doc in docs:
+                if doc._.trf_data is None:
+                    # TODO move to errors.py
+                    raise ValueError(
+                        "If the embedding layer is not updated during training, include it in 'annotating components'"
+                    )
```

Comment:
This can use E203 once https://github.com/explosion/spaCy/pull/11385 becomes available

Reply:
Ok, I'll update this once spaCy 3.5.0 is out.

Reply:
Done 🎉 .